agent_deploy/kubernetes/sysdig-agent-ebpf-daemonset-v1.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: sysdig-agent
+  labels:
+    app: sysdig-agent
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sysdig-agent
+    spec:
+      volumes:
+      - name: osrel
+        hostPath:
+          path: /etc/os-release
```

Comment:
@nathan-b , someone posted in the ebpf channel referring to this doc: https://github.com/draios/sysdig/wiki/eBPF pointing out that we already ask users to bind mount /etc from the host in to the sysdig container. given this, could we just make the agent do the same - i.e. bind mount /etc instead of /etc/os-release - for sake of consistency?

Reply:
If you think customers would be fine with exposing that much of their host filesystem into the container, we can totally make that change.